File: Arrays 101/01 What is an Array/Find all Numbers with Even Number of Digits.cs
using System;
using System.Collections.Generic;

public class Solution {
    public int FindNumbers(int[] nums) {
        int result = 0;

        foreach (int num in nums) {
            int count = 0;
            int tempNum = num;

            while (tempNum > 0) {
                tempNum /= 10;
                count++;
            } if (count % 2 == 0) {
                result++;
            }
        }
        return result;
    }
}

File: Arrays 101/01 What is an Array/Find all Numbers with Even Number of Digits.js
class Solution {
    findNumbers(nums) {
        let result = 0;

        nums.forEach(num => {
            let count = 0;
            let tempNum = num;

            while (tempNum > 0) {
                tempNum = Math.floor(tempNum / 10);
                count++;
            } if (count % 2 === 0) {
                result++;
            }
        });

        return result;
    }
}